features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/postgresql.sql

```diff
@@ -20,6 +22,7 @@ CREATE TABLE IDN_REMOTE_FETCH_REVISIONS (
 	DEPLOYED_DATE TIMESTAMP,
 	DEPLOYMENT_STATUS VARCHAR(255),
 	ITEM_NAME VARCHAR(255),
+	DEPLOYMENT_ERROR TEXT,
```

Comment:
can we change this field to, 
DEPLOY_ERR_LOG

Reasons,
1. It has a part LOG which says it is a log
2. Column name made short